tempodb/wal/wal.go

```diff
@@ -90,7 +89,7 @@ func (w *WAL) RescanBlocks(log log.Logger) ([]*AppendBlock, error) {
 		}
 
 		start := time.Now()
-		level.Info(log).Log("msg", "beginning replay", "file", f.Name(), "size", f.Size())
+		level.Info(log).Log("msg", "beginning replay", "file", f.Name())
```

Comment:
You can grab the file size using [`DirEntry.Info`](https://pkg.go.dev/io/fs#DirEntry) and [`FileInfo.Size`](https://pkg.go.dev/io/fs#FileInfo). Can you add the `size` label again? Then this PR will not change anything 🙂